docs/_templates/changelog/v3.0.0.md

```diff
@@ -30,6 +30,7 @@
 - <abbr title="New Features">✨</abbr> **[#279](https://github.com/mcbookshelf/Bookshelf/issues/279)** - Added the `#bs.block:play_block_sound` function for playing block sounds.
 - <abbr title="New Features">✨</abbr> **[#287](https://github.com/mcbookshelf/Bookshelf/issues/287)** - Updated `replace_type` to return whether a type was found and replaced.
 - <abbr title="Bug Fix">🐛</abbr> **[#320](https://github.com/mcbookshelf/Bookshelf/issues/320)** - Fixed functions that were unusable outside the Overworld.
+- <abbr title="Enhancements">⚡</abbr> **[#363](https://github.com/mcbookshelf/bookshelf/issues/363)** - `fill_block`, `fill_type`, `fill_random`, and `set_block`, `set_type` now accept `strict` for the `mode` parameter to specify whether the operation should trigger a block or shape updates.
```

Comment:
Since each changelog line starts with a verb in the past tense, we should maintain the same format (also should trigger a block or shape updates, i know this is mc changelog but it's super confusing for the shape part because it will actually replace blockstates and use the default one, i think we should only keep: accept new option strict, which places blocks as-is without block updates.)